### PR TITLE
Plugin variables

### PR DESCRIPTION
**Pull request by Fatme**

### CODE REVIEW

**Comment by justcodebuilduser** [comment]
:heart:

**Comment by justcodebuilduser** [comment]
:heart:

File: plugin-variables-helper.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+						convertedObject[`${propKey}.${innerPropKey}`] = innerPropValue;
+					});
+				}
+
+			});
+
+			return convertedObject;
+		}
+
+		return obj;
```
Comment (teobugslayer) [inline plugin-variables-helper.ts]: If we call this function with a number, it will return the number unchanged. Is this the proper behavior? Shouldn't we assert on non-object arguments?

**Comment by teobugslayer** [comment]
I'd write some unit tests on this code

File: plugin-variables-helper.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
+///<reference path=".d.ts"/>
+"use strict";
+
+export class PluginVariablesHelper implements IPluginVariablesHelper {
+	constructor(private $options: ICommonOptions) { }
+
+	/**
+	 * Checks if the specified CordovaPluginVariable exists in the --var option specified by user.
```
Comment (ligaz) [inline plugin-variables-helper.ts]: Is this Cordova related?

**Comment by Fatme** [comment]
We have unit tests but they are in icenium repo. I'll move them here :)

**Comment by justcodebuilduser** [comment]
:heart:

**Comment by justcodebuilduser** [comment]
:heart:

File: plugin-variables-helper.ts (hunk 1 above)
Comment (Fatme) [inline plugin-variables-helper.ts]: If we call this function with a number, it will return the passed number. I think this is the correct behavior.

**Comment by Fatme** [comment]
Actually the unit tests for this code in icenium-cli repo are integrated with the tests for plugins. I think we should separate the code but maybe I'll do it in another PR.

**Comment by Mitko-Kerezov** [comment]
:+1:
